### DISCUSSION TITLE
Add Walmart ONN 4k TV box

### DISCUSSION THREAD

**Discussion by Darin755** [zenfyrdev/bootloader-unlock-wall-of-shame#22] [Ideas]
https://xdaforums.com/t/guides-2023-walmart-onn-google-tv-4k-yoc.4610749/

This device used to be unlockable. However, thanks to a Google update we are screwed.

**Comment by havaianasdestruido** [comment]
Then could it be considered "Terrible" or "Impossible"?

**Comment by Lost-Entrepreneur439** [comment]
Onn is impossible to add, see #265 and #268
